app/models/container_template.rb

```diff
@@ -12,4 +12,55 @@ class ContainerTemplate < ApplicationRecord
   serialize :objects, Array
 
   acts_as_miq_taggable
+
+  def instantiate(params, project = nil)
```

Comment:
@bzwei `project` is the container project name in which the template will be instantiated. This container project is saved in the VMDB as you mentioed, here we just need to use its name. I can change this var name to `project_name`, it would probably be more accurate.